typing/ctype.ml

```diff
@@ -533,6 +533,25 @@ exception Non_closed of type_expr * bool
 let free_variables = ref []
 let really_closed = ref None
 
+(* [free_vars_rec] collects the generic variables of the input type
```

Comment:
I think "generic variables" is a poor choice of word.

Reply:
To give some color to my comment: when I see "generic" in the context of the typecheck my brain immediately jumps to "generic level". But here, we're happy to collect variables even if they are not at that level.

Reply:
Of course! I was reading code where we do only care about the generic variables, but this is clearly an error in my comment. Thanks, will fix.